contracts/src/v0.8/functions/dev/1_0_0/FunctionsRouter.sol

```diff
@@ -347,6 +347,10 @@ contract FunctionsRouter is IFunctionsRouter, FunctionsSubscriptions, Pausable,
       commitment.client
     );
 
+    if (err.length > 0) {
+      result.success = false;
```

Comment:
If we decide to change RequestProcessed then let's maybe call the fields "computationResult" and "computationError" to differentiate from USER_SUCCESS and USER_ERROR. ResultCode set to USER_ERROR should only indicate callback error - could be also renamed to clarify.